lib/services/northBound/deviceGroupAdministrationServer.js

```diff
@@ -161,7 +161,7 @@ function handleListDeviceGroups(req, res, next) {
             req.query.offset,
             listHandler);
     } else {
-        groupService.find(req.headers['fiware-service'], req.headers['fiware-servicepath'], listHandler);
+        groupService.find(req.headers['fiware-service'], req.headers['fiware-servicepath'], null, listHandler);
```

Comment:
What is supposed to be in ```null``` place?

Reply:
As you can see in https://github.com/telefonicaid/iotagent-node-lib/blob/master/lib/services/groups/groupService.js#L110, the third argument is the type, which is optional and not used by handleListDeviceGroups function.

You can see an example of using this function with type parameter in https://github.com/telefonicaid/iotagent-node-lib/blob/master/lib/services/groups/groupService.js#L257